.github/workflows/cont_integration.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         rust:
           - version: stable
             clippy: true
-          - version: 1.57.0 # MSRV
+          - version: 1.63.0 # MSRV
```

Comment:
This MSRV bump should be in a separate PR once we get the go-ahead from the LDK team.